docs/maintainer/knowledge_base.md

```diff
@@ -86,6 +86,13 @@ Some optional, but useful CMake options:
 - `-DCMAKE_FIND_FRAMEWORK=NEVER` and `-DCMAKE_FIND_APPBUNDLE=NEVER` Prevent CMake from using system-wide macOS packages.
 - `${CMAKE_ARGS}` Add variables defined by conda-forge internally. This is required to enable various conda-forge enhancements, like [CUDA builds](#cuda).
 
+:::warning
+
+As `${CMAKE_ARGS}` contains quoted elements in it, quoting `${CMAKE_ARGS}`
```

Comment:
The errors are because `CMAKE_ARGS` is a space separated list of options, not because it contains quoted elements.

Reply:
Ah, good to know given ([#general > ✔ Validation C/Fortran compatibility with CMAKE_ARGS @ 💬](https://conda-forge.zulipchat.com/#narrow/channel/457337-general/topic/.E2.9C.94.20Validation.20C.2FFortran.20compatibility.20with.20CMAKE_ARGS/near/524945021)). I'll fix this, but if I understand correctly, you're saying that the shell is then interpreting the expanded result of `"${CMAKE_ARGS}"` as a single argument instead of the multiple arguments it provides, correct?